Bottom: e853487120e022d657839d614cf6f91ade76bcad
Top:    2880507b6e5f57d436348f5c3fdb356880baf19e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-04-28 17:09:27 +0800

Refresh of vhost_net-prefetch-desc

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 25b6b3d..6e6225c 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -783,10 +783,10 @@ static void handle_rx_batched(struct vhost_net *net, struct vhost_log *vq_log)
 				return;
 
 			vhost_update_used_idx(vq, 1);
+			/* FIXME: batched signal */
+			vhost_signal(&net->dev, vq);
 			/* FIXME: count bytes */
 		}
-		/* FIXME: batched signal */
-		vhost_signal(&net->dev, vq);
 	}
 }
